src/ssl/ErrorDetailManager.cc

```diff
@@ -61,7 +62,8 @@ Ssl::ErrorDetailsList::getErrorDescr(Security::ErrorCode value)
 {
     const ErrorDetails::const_iterator it = theList.find(value);
     if (it != theList.end()) {
-        return it->second.descr.termedBuf();
+        SBuf rv(it->second.descr);
+        return rv.c_str();
```

Comment:
Returning a pointer to a temporary local variable. Please address all such bugs.

Reply:
I think using a static variable should be enough for that

Reply:
Using a static variable in such a context is unsafe because it silently screws up two callers in close proximity with each other. When possible, we should avoid such time bombs.

Reply:
What is the best way to do it? SBuf::copy() to a static buffer is very kludgy (e.g. it doesn't guarantee zero-termination)

Reply:
> I hope to find the answer to that question when I resume working on this PR.

I do not have a complete/polished answer, but its beginnings are sketched at https://github.com/squid-cache/squid/pull/1451#discussion_r1297800747.